eahub/templates/eahub/profile.html

```diff
@@ -1,295 +1,302 @@
+                                <h3 class="card__heading">Cause Area</h3>
+                            </div>
+                        </div>
+                        {% if not profile.has_cause_area_details %}N/A{% endif %}
+                        {% if profile.cause_areas or profile.cause_areas_other %}
+                            <div class="card__section">
+                                {{ profile.get_pretty_cause_areas|urlize|linebreaks }}
+                            </div>
+                        {% endif %}
+                        {% if profile.giving_pledges %}
+                            <div class="card__section">
+                                <div class="card__subtitle">Giving Pledges Taken</div>
+                                {{ profile.get_pretty_giving_pledges }}
+                            </div>
+                        {% endif %}
+                        {% if profile.available_to_volunteer %}
+                            <div class="alert alert-secondary">Available to volunteer</div>
+                        {% endif %}
+                        {% if request.user.id == profile.user_id %}
+                            <a class="btn btn-edit btn-outline-primary btn-sm" href="{% url 'edit_profile_cause_areas' %}">
+                                <i class="fa fa-pencil-alt"></i> Edit
+                            </a>
+                        {% endif %}
+                    </div>
+                </div>
             </div>
-        </a>
-      {% endif %}
 
-      {% load feature_flags %}
-      {% flag_enabled 'MESSAGING_FLAG' as messaging_flag %}
-      {% if request.user.id is not None and request.user.id != profile.user_id and profile.user.email is not None and messaging_flag %}
-        <a href="{% url 'message_profile' profile.slug  %}">
-          <div class="btn btn-default purple">
-            <span><i class="fa fa-pencil-alt"></i> Send Message</span>
-          </div>
-        </a>
-      {% endif %}
-    </div>
-  </div>
-</div>
+            <div class="col mb-7 mb-md-0">
+                <div class="card">
+                    <div class="card-body">
+                        <div class="row row-cols-2 card__header">
+                            <div class="col col-auto">
+                                <img class="card__img mb-2" src="{% static 'global/images/profile-detail/briefcase.webp' %}" loading="lazy">
+                            </div>
+                            <div class="col col-heading">
+                                <h3 class="card__heading">Career</h3>
+                            </div>
+                        </div>
+                        {% if not profile.has_career_details %}N/A{% endif %}
 
-<br><br>
+                        {% if profile.expertise_areas or profile.expertise_areas_other %}
+                            <div class="card__section">
+                                <div class="card__subtitle">Areas of expertise</div>
```

Comment:
Subtitles and body underneath have same styling, should be different  
![image](https://user-images.githubusercontent.com/15908208/108322119-fd285000-71bc-11eb-9217-1ec29a71942d.png)

Reply:
same, this is local cache issue